python/cudf/cudf/_lib/pylibcudf/libcudf/binaryop.pxd

```diff
@@ -5,7 +5,7 @@ from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
 from libcpp.string cimport string
 
-from cudf._lib.exception_handler cimport cudf_exception_handler
+from cudf._lib.pylibcudf.exception_handler cimport cudf_exception_handler
```

Comment:
I wonder if we should rename `cudf_exception_handler` to `libcudf_exception_handler`.

Reply:
Good point, will rename.